Resources/config/util.xml

```diff
@@ -10,6 +10,8 @@
 
         <service id="fos_user.util.user_manipulator" class="FOS\UserBundle\Util\UserManipulator">
             <argument type="service" id="fos_user.user_manager" />
+            <argument type="service" id="event_dispatcher" />
+            <argument type="service" id="service_container" />
```

Comment:
I'm not a big fan of injecting the whole container. Would there be another way to address the request?